### PR TITLE
[platform/mellanox] Integrate sai version 1.14 and sdk 4.3.2104

### PR DESCRIPTION
**Pull request by stephenxs**
**- What I did**
Integrate sai version 1.14 (commit 76ced9d4a147163098054a717ecce56d58478b35) and sdk 4.3.2104

clone sai from file:////auto/mswg/git/switchx/sai_interface.git SAI-Implementation
get sdk from http://arc-build-server/sdk/sx_sdk_eth-4.3.2104/SOURCES/

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/mellanox/fw.mk [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # mellanox firmware
 
-MLNX_FW_VERSION = 13.2000.1420
+# Stephen Sun comments out for sdk/sai integration
+# MLNX_FW_VERSION = 13.2000.1420
+#
+MLNX_FW_VERSION = 13.2000.2104
+
 MLNX_FW_FILE = fw-SPC-rel-$(subst .,_,$(MLNX_FW_VERSION))-EVB.mfa
```
Comment (keboliu) [inline platform/mellanox/fw.mk:9]: the url should be changed since it not upstreamed yet.

File: platform/mellanox/mlnx-sai/Makefile [hunk 2]
```diff
@@ -7,7 +7,14 @@ MAIN_TARGET = mlnx-sai_1.mlnx.$(MLNX_SAI_VERSION)_amd64.deb
...
 	rm -rf SAI-Implementation
-	git clone https://github.com/Mellanox/SAI-Implementation.git
+#
+#	Stephen Sun comment this out for new SAI integration
+#
+#	git clone https://github.com/Mellanox/SAI-Implementation.git
+#
+#	It should be cloned from our internal git as following
+#
+	git clone file:////auto/mswg/git/switchx/sai_interface.git SAI-Implementation
```
Comment (keboliu) [inline platform/mellanox/mlnx-sai/Makefile:17]: we need specific commit.

File: platform/mellanox/fw.mk (hunk 1 above)
Comment (stephenxs) [reply platform/mellanox/fw.mk:9]: fixed.

File: platform/mellanox/mlnx-sai/Makefile (hunk 2 above)
Comment (stephenxs) [reply platform/mellanox/mlnx-sai/Makefile:17]: it is defined in the code below.

**Comment by stephenxs** [comment]
This PR is based on an out-of-date commit of 201811 branch, so I will close it and raise another one.

**Pull request closed without merging**